app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardResultsStep/MappingWizardResultsStep.js

```diff
@@ -4,42 +4,64 @@ import { noop, Button, Spinner } from 'patternfly-react';
 
 class MappingWizardResultsStep extends React.Component {
   componentDidMount() {
-    const { postMappingsUrl, postTransformMappingsAction, transformationsBody } = this.props;
+    const {
+      postMappingsUrl,
+      postTransformMappingsAction,
+      transformationsBody,
+      editingMapping,
+      updateTransformationMappingAction
+    } = this.props;
 
-    postTransformMappingsAction(postMappingsUrl, transformationsBody);
+    if (editingMapping) {
+      updateTransformationMappingAction(postMappingsUrl, editingMapping.id, transformationsBody);
```

Comment:
If this `transformationsBody` is exactly identical to it's initial state, maybe we should not make a backend edit call. What do you think?
Sometimes users may use the wizard to just browse through things and not necessarily make a change.

@mturley This is also applicable to your Edit Plan PR.

Reply:
It seems the `transformation_mapping_items` arrays are quite different between `transformationBody` and `editingMapping`

Example of transformation item from `editingMapping`
```
{
  created_at: "2018-09-26T00:44:22Z"
  destination_id: "1"
  destination_type: "EmsCluster"
  id: "6"
  options: {merge: ƒ, replace: ƒ, without: ƒ, asMutable: ƒ, set: ƒ, …}
  source_id: "3"
  source_type: "EmsCluster"
  transformation_mapping_id: "1"
  updated_at: "2018-09-26T00:44:22Z"
}
```

Example of transformation item from `transformationBody`
```
{
  destination: "/api/clusters/1"
  source: "/api/clusters/3"
}
```

It will require a little more code, but I can make this happen.  What would you guys like for me to do?  @AparnaKarve @mturley

Reply:
I would say let's not make the API edit call, if we don't have to.

This is in fact a use case that is consistently handled in ui-classic in other screens -- you will notice that the 'Save' button does not light up unless the form is truly dirty (i.e. different before and after values)

Reply:
Resolved by b39135e4c033c92148298ee11b71b3df855bf0eb